File: dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/AdaptiveCards/AdaptiveCardsOptions.cs
namespace Microsoft.Teams.AI
{
    /// <summary>
    /// Options for AdaptiveCards class.
    /// </summary>
    public class AdaptiveCardsOptions
    {
        /// <summary>
        /// Data field used to identify the Action.Submit handler to trigger.
        /// </summary>
        /// <remarks>
        /// When an Action.Submit is triggered, the field name specified here will be used to determine
        /// the handler to route the request to.
        /// Defaults to a value of "verb".
        /// </remarks>
        public string? ActionSubmitFilter { get; set; }
    }
}

File: python/packages/ai/teams/adaptive_cards/adaptive_cards_options.py
"""
Copyright (c) Microsoft Corporation. All rights reserved.
Licensed under the MIT License.
"""

from __future__ import annotations

from dataclasses import dataclass


@dataclass
class AdaptiveCardsOptions:
    action_submit_filer: str = "verb"
    """
    Data field used to identify the Action.Submit handler to trigger.
    When an Action.Submit is triggered, the field name specified here will be used to determine
    the handler to route the request to.
    """
